TESTING.md

```diff
@@ -0,0 +1,9 @@
+Trailblazer-Rails support rails 5.2.0 and up.
```

Comment:
What's the benefit of having TESTING.md ? I think we should have some reference in README.md to easily identify.

Reply:
TESTING is a standard file name for testing instructions. Like CHANGELOG, AUTHORS, ect

Reply:
Yeah yeah, but should we mention this in README as well ?

Reply:
I don't think so . We are not referencing the other files either.